go-controller/pkg/ovn/service.go

```diff
@@ -512,3 +512,27 @@ func svcQualifiesForReject(service *kapi.Service) bool {
 	_, ok := service.Annotations[OvnServiceIdledAt]
 	return !(config.Kubernetes.OVNEmptyLbEvents && ok)
 }
+
+// SVC can be of types 1. clusterIP, 2. NodePort, 3. LoadBalancer,
+// or 4.ExternalName In this scenario we will only address 1.
+// TODO adjust for upstream patch when it lands:
+// https://bugzilla.redhat.com/show_bug.cgi?id=1908540
+func getSvcVips(service *kapi.Service) []net.IP {
+	klog.V(5).Infof("Service %s is applied to same namespace as network Policy, finding cluster IPs", service.Name)
+	//If service is type ClusterIP, VIP will always be that IP
+	if util.ServiceTypeHasClusterIP(service) {
+		klog.V(5).Infof("Service %s is of type kapi.ServiceTypeClusterIP || kapi.ServiceTypeNodePort || kapi.ServiceTypeLoadBalancer", service.Name)
```

Comment:
remove this logging, it doesnt really provide any value. Instead just log in the caller if you found/adding an IP to the address set from this.